### PR TITLE
module: add pif.json dependencies

### PR DESCRIPTION
**Pull request by saadelasfur14**
"Copy default pif.json if it doesn't exist."
"Copy default pif.json over the old one."

important to avoid issues when applying new pif.json file

### CODE REVIEW

File: module/customize.sh [hunk 1]
```diff
@@ -36,3 +36,15 @@ if [ -d "/system/app/EliteDevelopmentModule" ]; then
...
+
+# Copy default pif.json if it doesn't exist.
+if [ ! -e /data/adb/pif.json ]; then
+	cp -af $MODPATH/pif.json /data/adb/pif.json
+	ui_print "- Moved default pif.json file."
+fi
+
+# Copy default pif.json over the old one.
+if [ ! -d /data/adb/pif.json ]; then
+	cp -af $MODPATH/pif.json /data/adb/pif.json
```
Comment (vladrevers) [inline module/customize.sh:48]: Right now, in the pif.json module, it's just a data-independent template.
Even assuming there will be some data there, why would you want to overwrite a manually found user fingerprint (which lives for months) with a regular fingerprint (which locks up after a couple weeks)?

**Comment by chiteroman** [comment]
The code do the same...

**Comment by saadelasfur14** [comment]
> The code do the same...

do the code applies new pif.json on the exist one? lol

**Comment by saadelasfur14** [comment]
Issue Solved

**Pull request closed without merging**